ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/manage-redirects/redirect-row/redirect-row.html

```diff
@@ -9,13 +9,17 @@
       ${properties.target}
    </td>
    <td is="coral-table-cell" class="statusCode" data-value="${properties.statusCode}" style="width:50px;">${properties.statusCode}</td>
+   <td is="coral-table-cell" class="createdBy"
```

Comment:
there is an inconsistency in the use of the model class for the existing fields and the new ```createdBy``` field. Can you harmonize them?

Reply:
I refactored the code to use the model class across the component.